bedrock/firefox/templates/firefox/sync.html

```diff
@@ -34,15 +34,18 @@ <h2 class="c-sub-navigation-title">
 {% endblock %}
 
 {% block content %}
-
-{% call hero(
-  title=ftl('firefox-sync-sync-your-firefox'),
-  class='mzp-has-image mzp-t-product-firefox',
+  {% call split(
   image_url='img/firefox/accounts/trailhead/value-knowledge.jpg',
   include_highres_image=True,
-  include_cta=True,
-  heading_level=1
-  ) %}
+  media_after=True,
+  block_class='mzp-l-split-center-on-sm-md',
+  media_class='mzp-l-split-h-center'
+) %}
+<div class="mzp-c-logo mzp-t-logo-xl mzp-t-product-firefox">
+  Firefox Browser
```

Comment:
Just realizing, this should probably be a string and added to the Fluent file. Sorry I missed that.
You can add it to `/l10n/en/firefox/sync.ftl` and use the `{ -brand-firefox-browser }` placeable.